lalrpop-snap/Cargo.toml

```diff
@@ -26,6 +26,7 @@ regex = "0.2.1"
 regex-syntax = "0.4.0"
 petgraph = "0.4.4"
 rustc-serialize = "0.3"
+string_cache = "0.7.1"
```

Comment:
As an aside, I've been mildly regretting moving to `string_cache` and wondering if one of the other alternatives on crates.io. would've been better. In particular https://crates.io/crates/internment seems to offer `Copy` tokens (at the price of never freeing, which I doubt we care about). Anyway.